pkgs/development/python-modules/django-cors-headers/default.nix

```diff
@@ -2,30 +2,34 @@
   lib,
   fetchFromGitHub,
   buildPythonPackage,
+  setuptools,
   django,
   pytest-django,
   pytestCheckHook,
 }:
 
 buildPythonPackage rec {
   pname = "django-cors-headers";
-  version = "3.13.0";
-  format = "setuptools";
+  version = "4.4.0";
+  pyproject = true;
+  build-system = [ setuptools ];
 
   src = fetchFromGitHub {
     owner = "adamchainz";
-    repo = "django-cors-headers";
+    repo = pname;
```

Comment:
I believe `repo = "django-cors-headers";` is still preferred, athough the two have no difference under `buildPythonPackage rec`.

Reply:
> You can also send changes to existing packages in separate PRs for faster reviews and merges.

Makes sense, I'm working on that

> I believe repo = "django-cors-headers"; is still preferred, athough the two have no difference under buildPythonPackage rec.

I see this pattern in more than four thousand places all over Nixpkgs, are you sure? Not important for me though.

Reply:
Not important to me either.

Reply:
See https://github.com/nix-community/nixpkgs-lint/issues/21